### PR TITLE
Initial commit of Security Headers Cheat Sheet

### PR DESCRIPTION
**Pull request by agektmr**
Initial commit of the security headers cheat sheet draft.

This draft does **NOT** need reviews until we are ready.

### CODE REVIEW

**Comment by agektmr** [comment]
This is still WIP. Please do not review yet.

**Comment by agektmr** [comment]
cc: @arturjanc @lweichselbaum @maudnals

**Comment by agektmr** [comment]
@mihajlija  I'd like to change the URL to `web.dev/security-headers` as the title has changed from original. (And I like `/security-headers` rather than `/security-headers-quick-reference`). Also, we'll add a redirect from `/security-headers-cheat-sheet` to there before merging. Just so you know.

File: src/site/content/en/secure/security-headers-cheat-sheet/index.md [hunk 1]
```diff
@@ -0,0 +1,1020 @@
...
+Security headers recommended for all websites:
+: [X-Content-Type-Options](#xcto)
+: [X-Frame-Options](#xfo)
+: [Cross-Origin Resource Policy (CORP)](#corp)
+: [Cross-Origin Opener Policy (COOP)](#coop)
+: [HTTP Strict Transport Security (HSTS)](#hsts)
+
+Security headers recommended for websites that handle sensitive user data:
+: [Content Security Policy (CSP)](#csp)
+: [Trusted Types](#tt)
```
[the remark this answers is not in the document]
Comment (agektmr) [reply src/site/content/en/secure/security-headers-cheat-sheet/index.md:26]: The order was swapped based on feedback. Let's revisit this at the last minute.

**Comment by mihajlija** [comment]
@agektmr Is there a specific reason to use redirect, rather than just change the URL? Since the PR hasn't been merged yet, changing the folder name will do the trick without the need for redirecting.

**Comment by agektmr** [comment]
An I/O slide includes the URL.

**Comment by mihajlija** [comment]
@agektmr Thanks for clarifying!

**Review by mihajlija — APPROVED** [review]

**Pull request merged**